Modules/Devices/src/Devices.Application/Users/Commands/SeedTestUsers/Handler.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Backbone.Modules.Devices.Application.Infrastructure.Persistence.Database;
+using Backbone.Modules.Devices.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Devices.Domain.Entities;
+using Enmeshed.DevelopmentKit.Identity.ValueObjects;
+using Enmeshed.Tooling;
+using MediatR;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+
+namespace Backbone.Modules.Devices.Application.Users.Commands.SeedTestUsers;
+public class Handler : IRequestHandler<SeedTestUsersCommand, SeedTestUsersResponse>
+{
+    private readonly IPasswordHasher<ApplicationUser> _passwordHasher = new PasswordHasher<ApplicationUser>();
```

Comment:
Can you try to let the DI container inject the `IPasswordHasher<ApplicationUser>`? I would expect it to work.